src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+html {
+  border: none;
+  outline: none;
```

Comment:
html haven't borders and outlines by default), so remove it

Reply:
done